spec/integration_spec.rb

```diff
@@ -271,4 +271,71 @@ def parse_html(html)
     expect(result).to include("<td>One</td><td>1</td>")
     expect(result).to include("<td>Two</td><td>2</td>")
   end
+
+  it "doesn't inline styles in media queries with features" do
+    document = Roadie::Document.new <<-HTML
+      <html>
+        <head>
+          <link rel="stylesheet" href="/style.css">
+        </head>
+        <body>
+          <div class="colorful"></div>
+        </body>
+      </html>
+    HTML
+
+    document.asset_providers = TestProvider.new(
+      "/style.css" => <<-CSS
+        .colorful { color: green; }
+        @media screen and (max-width 600px) {
+          .colorful { color: red; }
+        }
+      CSS
+    )
+
+    result = Nokogiri::HTML.parse document.transform
```

Comment:
NB parse_html caused a failure 🤷‍♂️ 
https://travis-ci.org/Mange/roadie/jobs/376625294#L554

Reply:
This was resolved with subsequent commits no?

Reply:
Not satisfactorily. I don't understand why other tests can use the helper method parse_html but my test fails

Reply:
Oh right, I see it now.

It's the scope of the block. You hadn't defined the `parse_html` function in block scope you are working on.